[5.7] Use Ubuntu 16.04

carusogabriel opened the pull request:
<!--
Pull Requests without a descriptive title, thorough description, or tests will be closed.

Please include the benefit to end users; the reasons it does not break any existing features; how it makes building web applications easier, etc.
-->

Ubuntu 16.04 is already available for usage: https://docs.travis-ci.com/user/reference/xenial/#using-xenial.

carusogabriel committed:
Use Ubuntu 16.04

ankurk91 commented:
Should also remove the sudo false

https://github.com/laravel/framework/blob/d8b860f3ed602645dee401f3ae43d21d08dd7ee6/.travis.yml#L20

GrahamCampbell commented:
Hmm, these builds are running much slower than the older ubuntu.

carusogabriel commented:
@ankurk91 Any reasons why?

@GrahamCampbell I've noticed that, but as was only 1 build, I don't think we can judge :confused:

ankurk91 commented:
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

taylorotwell commented:
Why wouldn't we use 18.04?

carusogabriel commented:
@taylorotwell Travis, unfortunately, doesn't have it. I'll update this patch with @ankurk91's suggestion, and it should be ready to go.

carusogabriel committed:
Remove sudo configuration from Travis CI

This will became not available in the future: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

driesvints commented:
Build speed seems to be ok now.
